### ISSUE TITLE
Stage1: Train 2D pose only

### ISSUE THREAD

**Issue by NguyenDangBinh** [xingyizhou/pytorch-pose-hg-3d#57]
dear,
It can be run on GPU, not CPU?

**Comment by xingyizhou** [comment]
Yes, I haven't tested training with CPU. And it is not recommended to do so since it would be too slow. However if you really want to try, you can change this line https://github.com/xingyizhou/pytorch-pose-hg-3d/blob/master/src/main.py#L38 to `opt.device = torch.device('cpu')`
